browser/ui/views/brave_actions/brave_actions_container.h

```diff
@@ -85,6 +85,8 @@ class BraveActionsContainer : public views::View,
   // views::View:
   void ChildPreferredSizeChanged(views::View* child) override;
 
+  bool ExecuteAction(const std::string& id, bool by_user);
```

Comment:
Instead of adding public method for testing purpose, how about declaring test class as a friend class?
With that, we don't need to add method and test code can use |actions_| directly.

Reply:
Good idea, will do!